### DISCUSSION TITLE
Order Delivery Date Support [done] ✅

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5052] [Working on Features]
Integrate with Order Delivery Date Plugin: [https://www.tychesoftwares.com/store/premium-plugins/order-delivery-date-for-woocommerce-pro-21/](https://www.tychesoftwares.com/store/premium-plugins/order-delivery-date-for-woocommerce-pro-21/)

**Comment by HarrySmith01** [comment]
Instead of integrating this one, Below will be more better:

https://iconicwp.com/products/woocommerce-delivery-slots/
https://codecanyon.net/item/woocommerce-delivery/26548021

Above one not only enable delivery date but enables delivery time slot too. Delivery time slot while good for large items like TV, Refrigerators delivery but it is also being used for Grocery delivery services.

**Comment by minhcasi** [comment]
"I want to add the feature where the customers can select the delivery time from the check out  just like in this woo plugin :

https://woocommerce.com/products/woocommerce-order-delivery/

Please suggest how can we implement this feature."

**Comment by minhcasi** [comment]
Hi friends, just sharing that our team is working on this feature and put heavy development effort to release it in the next couple of weeks 👍

**Reply by faheemjavid131** [reply]
We are waiting sir

**Reply by ekramul-1** [reply]
You guys are really amazing & rocking. God bless you guys. We all always wait for new features.

**Comment by CeoAlexMartinez** [comment]
Esperando esta caracteristica

**Comment by smark98** [comment]
Hi,

I would like to ask which delivery date and time plugin will be integrate? Because I would like to purschase that plugin to set up my page sucessfully. 

I see the plugin which you integrated to delivery boy app is also have delivery time options.
https://wclovers.com/product/woocommerce-frontend-manager-delivery/

Or will be works with both plugin? 
https://woocommerce.com/products/woocommerce-order-delivery/ 
And
https://wclovers.com/product/woocommerce-frontend-manager-delivery/
?

**Reply by Haniel5** [reply]
Hi, we will use this plugin https://wordpress.org/plugins/order-delivery-date-for-woocommerce/  - Thank you!

**Comment by gerlandog** [comment]
Hello, I would like to know estimated release time for this feature, I was told when I bought that it would be ready by the end of may....
thanks!
